octodns/provider/ovh.py

```diff
@@ -9,9 +9,9 @@
 
 logger = getLogger('Ovh')
 try:
-    logger.warn('octodns_ovh shimmed. Update your provider class to '
-                'octodns_ovh.OvhProvider. '
-                'Shim will be removed in 1.0')
+    logger.warning('octodns_ovh shimmed. Update your provider class to '
+                   'octodns_ovh.OvhProvider. '
+                   'Shim will be removed in 1.0')
```

Comment:
no big deal, but was this intentional for this PR?

Reply:
Tbh, not sure how it made it through the ovh PR without getting noticed. Maybe warnings are errors. Will look into that. It was just in here to avoid a dedicated PR since I've been doing so many of them already

Reply:
Yeah those are warnings not errors.